tests/test_visitors/test_ast/test_statements/test_variable_used_outside_of_block.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    VariableUsedOutsideOfForBlockViolation,
+    VariableUsedOutsideOfWithBlockViolation
+)
+from wemake_python_styleguide.visitors.ast.statements import (
+    VariableUsedOutsideOfBlockVisitor,
+)
+
+variable_used_outside_with = """
+def function():
+    with a.open() as x:
+        some()
```

Comment:
These calls do no make any difference. You can just drop them and use `...` instead.